pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -5523,17 +5526,19 @@
     "System.Text.Json": {
       "StableVersions": [
         "4.6.0",
-        "4.7.0"
+        "4.7.0",
+        "4.7.1"
       ],
       "BaselineVersion": "4.7.0",
       "InboxOn": {
         "netcoreapp3.0": "4.0.0.0",
-        "netcoreapp3.1": "4.0.1.0",
+        "netcoreapp3.1": "4.0.1.1",
```

Comment:
For future reference, this change was technically wrong as this assembly is also shipping out of an OOB package. When that is the case, you want to keep the InboxOn version frozen to the version you first shipped it on that framework, that way, if you ever service any package from this branch that takes a dependency on this library, our packaging infrastructure will force adding a package reference to the serviced version of this package.

cc: @Anipik @ericstj

Reply:
In all cases InboxOn should be frozen.  Once an assembly is inbox in a framework that's defined the minimum version provided by the framework.  Since a framework is targeted with a single thing: the TFM, we cannot depend on anything but the minimum version.